packages/manager/apps/web-domains/public/translations/domain/Messages_fr_FR.json

```
@@ -136,6 +169,18 @@
   "domain_tab_general_information_associated_subdomains_multisite_content": "Aucun sous-domaine ou multisite",
   "domain_tab_general_not_suspended": "Actif",
   "domain_tab_general_suspended": "Suspendu",
+  "domain_status_technical_suspended": "Procédure en cours",
+  "domain_status_pending_installation": "En cours d'installation",
+  "domain_status_auto_renew_in_progress": "Renouvellement en cours",
+  "domain_tab_general_information_status_enabled": "Activée",
```

Is it for the protection against transfer?
- If yes:
```suggestion
  "domain_tab_general_information_protection_against_transfer_enabled": "Activée",
```
- If no: 
```suggestion
  "domain_tab_general_information_status_enabled": "Activé",
```
I think that generically, the enabled status should be spelled "Activé", I think that "Activée" is very specific (only for protection against transfer)